frontend/js/src/common/brainzplayer/BrainzPlayer.tsx

```diff
@@ -783,13 +813,6 @@ export default function BrainzPlayer() {
     setDurationMs(newDurationMs);
   };
 
-  // Effect to start the timer when durationMs changes
-  React.useEffect(() => {
-    if (durationMs > 0) {
-      startPlayerStateTimer();
-    }
-  }, [durationMs, startPlayerStateTimer]);
-
```

Comment:
I'm not exactly sure why we had this mechanism, but I suspect it was because we don't have a *good* mechanism to detect that playback has actually started, once we ask a datasource to play.
However it is not a solid way to do so, as we can receive a duration without playback having actually started, causing for example [LB-1182](https://tickets.metabrainz.org/browse/LB-1182).

Opting to keep only the `startPlayerStateTimer()` call in playerPauseChange (line 802).